src/hotspot/share/jvmci/jvmciCodeInstaller.cpp

```diff
@@ -1331,12 +1184,6 @@ void CodeInstaller::site_DataPatch(CodeBuffer& buffer, jint pc_offset, JVMCIObje
       if (!_immutable_pic_compilation) {
```

Comment:
All _immutable_pic_compilation mentions can be removed as well. It is true only for AOT compiles produced by Graal. It's never going to be used without AOT.

Reply:
Done. I removed _immutable_pic_compilation in JVMCI in Hotspot.